### DISCUSSION TITLE
How to build and use resampler?

### DISCUSSION THREAD

**Discussion by blagojco-juca** [google/oboe#1153] [Q&A] [answered]
We need a solution that will resample audio (from live Zoom call) sample rate from 32000 to 44100. Looks like the resampler can do this. We have ByteBuffer from the ZoomSDK and we just need to resample. We are beginners with c++. Is there already prepared, built version of the Oboe Resampler that we can just add to our project? Or is there ready CmakeList.txt file that will build the code into .so files? Any idea or approach can help

Thanks

**Comment by philburk** [comment, marked as answer]
Yes, the resampler can be used in other projects outside of Oboe.

We do not provide any pre-built libraries or build tools. But I think adding a CMakeLists.txt file would be good.

I added some instructions for building. It's pretty easy:

https://github.com/google/oboe/blob/master/src/flowgraph/resampler/README.md#building-the-resampler
